audit-cli/README.md

```diff
@@ -515,6 +622,151 @@ include             : 3 files, 4 usages
+Total unique procedures: 36
+Total procedure appearances: 93
+```
+
+**With `--list-summary`:**
+```
+File: path/to/file.rst
+Total unique procedures: 36
+Total procedure appearances: 93
+
+Unique Procedures:
+  1. Before You Begin
+  2. Install MongoDB Community Edition
+  3. Configuration
+  4. Run MongoDB Community Edition
+  ...
+```
+
+**With `--list-all`:**
+```
+File: path/to/file.rst
+Total unique procedures: 36
+Total procedure appearances: 93
+
+================================================================================
+Procedure Details
+================================================================================
+
+1. Before You Begin
+   Line: 45
+   Implementation: procedure-directive
+   Steps: 5
+   Contains sub-procedures: no
+   Appears in 2 selection(s):
+     - docker, None, None, None, None, None, without-search-docker
+     - docker, None, None, None, None, None, with-search-docker
+
+   Steps:
+     1. Pull the MongoDB Docker Image
+     2. Run the MongoDB Docker Container
+     3. Verify MongoDB is Running
+     4. Connect to MongoDB
+     5. Stop the MongoDB Docker Container
+
+2. Install MongoDB Community Edition
+   Line: 123
+   Implementation: ordered-list
+   Steps: 4
+   Contains sub-procedures: yes
+   Appears in 10 selection(s):
+     - linux, None, None, tarball, None, None, with-search
+     - linux, None, None, tarball, None, None, without-search
+     ...
+
+   Steps:
+     1. Download the tarball
+     2. Extract the files from the tarball
+     3. Ensure the binaries are in a directory listed in your PATH
+     4. Run MongoDB Community Edition
+```
+
+**Understanding the Counts:**
+
+The command reports two key metrics:
+
+1. **Total unique procedures**: Number of distinct procedures (grouped by heading and content hash)
+   - Procedures with the same heading but different content are counted separately
+   - Procedures with identical content are counted once, even if they appear in multiple selections
+
+2. **Total procedure appearances**: Total number of times procedures appear across all selections
+   - If a procedure appears in 5 different selections, it contributes 5 to this count
+   - This represents the total number of procedure instances a user might encounter
+
+**Example:**
+- A file might have **36 unique procedures** that appear a total of **93 times** across different selections
+- This means some procedures appear in multiple selections (e.g., a "Before You Begin" procedure that's the same for Docker with and without search)
+
+**Supported Procedure Types:**
+
+The command recognizes:
+- `.. procedure::` directives with `.. step::` directives
+- Ordered lists (numbered or lettered) as procedures
+- `.. tabs::` directives with `:tabid:` options for variations
+- `.. composable-tutorial::` directives with `.. selected-content::` blocks
+- Sub-procedures (ordered lists within steps)
+- YAML steps files (automatically converted to RST format)
+
+**Deterministic Parsing:**
```

Comment:
might be useful to link out to the PROCEDURE_PARSING doc somewhere; i found it answered a few questions that popped in my head as i was reading this README